src/helpers.test.js

```diff
@@ -6,95 +6,172 @@ import {
     getDefaultValueForType,
     getDefaultValueFromOptions,
     getFunctionName,
+    removeFirstKeyIfNumber,
+    getErrorIndexFromKeys,
 } from './helpers';
 
 describe('helpers', () => {
-    it('should create object from another object with specific keys', () => {
-        const bar = {
-            foo: 'bar',
-            foo2: 'bar2',
-            foo3: 'bar3',
-            foo4: 'bar4',
-        };
-        const foo = pick(bar, ['foo', 'foo2']);
+    describe('pick', () => {
+        it('should create object from another object with specific keys', () => {
+            const bar = {
+                foo: 'bar',
+                foo2: 'bar2',
+                foo3: 'bar3',
+                foo4: 'bar4',
+            };
+            const foo = pick(bar, ['foo', 'foo2']);
 
-        expect(foo).toEqual({
-            foo: 'bar',
-            foo2: 'bar2',
+            expect(foo).toEqual({
+                foo: 'bar',
+                foo2: 'bar2',
+            });
         });
     });
+    describe('difference', () => {
+        it('should compere two array and return difference', () => {
+            const foo = ['foo', 'foo2', 'foo3', 'foo4'];
+            const bar = ['foo', 'foo2'];
```

Comment:
These two array does not show whether the difference is **calculated** by keys, or by values.

Reply:
I will work on it next time.